.github/workflows/docker-builds.yaml

```
@@ -60,6 +63,20 @@ jobs:
             latest=false
             suffix=${{ matrix.flavor }}
 
+      - name: Generate tags for prefecthq/prefect
+        id: metadata-prod
+        uses: docker/metadata-action@v4
+        with:
+          images: prefecthq/prefect,enable=${{ github.event_name == 'release' }}
```

Might be worth highlighting this is what prevents this from running on development builds.

Should we cover `prereleased` here as well? I'm not sure if we have a use-case for that.

---

both `released` and `prereleased` event types will pass this conditional (checking which one of both it is, is another topic)

how is this?
```suggestion
          # only generate the production tags on release events
          images: prefecthq/prefect,enable=${{ github.event_name == 'release' }}
```

---

👍 

Slightly different note: We'll not want to apply the `latest` tag for prereleases.